test/Atmos/EDMF/stable_bl_edmf_incompressible.jl

```diff
@@ -0,0 +1,256 @@
+using JLD2, FileIO
+using ClimateMachine
+ClimateMachine.init(;
+    parse_clargs = true,
+    output_dir = get(ENV, "CLIMATEMACHINE_SETTINGS_OUTPUT_DIR", "output"),
+    fix_rng_seed = true,
+)
+using ClimateMachine.SingleStackUtils
+using ClimateMachine.Checkpoint
+using ClimateMachine.BalanceLaws: vars_state
+import ClimateMachine.BalanceLaws: projection
+using ClimateMachine.Atmos
+const clima_dir = dirname(dirname(pathof(ClimateMachine)));
+import CLIMAParameters
+
+include(joinpath(clima_dir, "experiments", "AtmosLES", "stable_bl_model.jl"))
+include("edmf_model.jl")
+include("edmf_kernels.jl")
+
+projection(
+    ::AtmosModel,
+    ::TendencyDef{Flux{O}, PV},
+    x,
+) where {O, PV <: Momentum} =
+    x .* SArray{Tuple{3, 3}}(1, 1, 1, 1, 1, 1, 0, 0, 0)
+
+projection(::AtmosModel, ::TendencyDef{Source, PV}, x) where {PV <: Momentum} =
+    x .* SVector(1, 1, 0)
```

Comment:
@kpamnany, @mwarusz, @LenkaNovak Here's how we're doing the projections for a single experiment. I'm not sure the best way to incorporate this into the model itself. We could add a dispatch mechanism inside the AtmosModel so that we have default projection methods for each model, but I'm not sure that putting a dispatch mechanism inside the AtmosModel is the appropriate place.

Reply:
This has moved to `src/Atmos/Model/projections.jl`